skillmap/src/App.tsx

```diff
@@ -358,9 +358,11 @@ class AppImpl extends React.Component<AppProps, AppState> {
     protected async initColorThemeAsync() {
         // Load theme colors
         const prefThemeId = await authClient.getColorThemeIdAsync();
-        let initialTheme = this.props.highContrast ?
-                    pxt.appTarget?.appTheme?.highContrastColorTheme :
-                    prefThemeId ?? pxt.appTarget?.appTheme?.defaultColorTheme;
+        let initialTheme = this.props.highContrast
+            ? pxt.appTarget?.appTheme?.highContrastColorTheme
+            : prefThemeId && this.themeManager.isKnownTheme(prefThemeId)
```

Comment:
```suggestion
            : (prefThemeId && this.themeManager.isKnownTheme(prefThemeId))
```

nitpick: i always forget the operator precedence for ternarys, so i tend to put in parentheses to clarify things